app/app-config/baseConfig.js

```diff
@@ -1,8 +1,11 @@
 define([], function() {
 /* eslint-enable max-statements */
     var baseURL = 'https://webpush-you-host.mobifydemo.io';
-    // Line altered by generator. If changing please change generator as well!!
+
+    // Lines altered by generator. If changing please change generator as well!!
     var useTabLayout = false;
+    var previewEnabled = false;
+    var previewBundle = '<preview_bundle>';
```

Comment:
maybe add a `// end lines altered by generator` here just so people for sure know what is/isn't safe to edit without touching the generator